notebooks/242-freevc/242-freevc.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    onnx_path = Path(OUTPUT_DIR / (BASE_MODEL_NAME + "_fp32")).with_suffix(".onnx")
</pre><p><code>onnx_path </code> is reused across different models, shall we separate them by different name ?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>